tensorrt_llm/_torch/pyexecutor/_util.py

```
@@ -478,6 +481,13 @@ def _create_kv_cache_manager(
             )
         # KVCacheManager (Non-draft) modifies the max_seq_len field, update it to self
         if model_engine.kv_cache_manager_key == ResourceManagerType.KV_CACHE_MANAGER:
+            executor_config.max_seq_len = kv_cache_manager.max_seq_len
```

Ruff (0.12.2)

Undefined name `executor_config`

(F821)